### ISSUE TITLE
Bug: Fail to create VPC

### ISSUE THREAD

**Issue by matsev** [hupe1980/cdk-multi-profile-plugin#5]
Credentials fail when attempting to create a VPC

**Step by step:**

1. Create and add a network stack (se below) to your AWS CDK app:

    ```typescript
    import { Construct, Stack, StackProps } from '@aws-cdk/core';
    import { SubnetType, Vpc } from '@aws-cdk/aws-ec2';
    
    export class NetworkStack extends Stack {
      public readonly vpc: Vpc;
    
      constructor(scope: Construct, id: string, props?: StackProps) {
        super(scope, id, props);
    
        this.vpc = new Vpc(this, 'VPC', {
          subnetConfiguration: [{
            name: 'Isolated',
            cidrMask: 24,
            subnetType: SubnetType.ISOLATED,
          }]
        });
      }
    }
    ```

2. Build and deploy:
    ```bash
    $ npm run build
    $ npx cdk synth
    ```

----

**Expected result**
The AWS CDK should deploy the app and create a new VPC

----

**Actual result**
```bash
[Error at network-stack] Need to perform AWS calls for account 112233445566, but no credentials found. Tried: default credentials.
```

----
`package.json` excerpt:
```json
{
  "name": "my-app-with-vpc",
  "private": true,
  "bin": {
    "cdk": "bin/cdk.js"
  },
  "scripts": {
    "build": "tsc",
  },
  "devDependencies": {
    "@aws-cdk/aws-ec2": "^1.23.0",
    "@aws-cdk/core": "^1.23.0",
    "cdk-multi-profile-plugin": "^1.1.2",
    "ts-node": "^8.1.0",
    "typescript": "~3.7.2"
  }
}
```

`cdkmultiprofileplugin.json` excerpt:
```json
{
  "awsProfiles": {
    "112233445566": "dev-mfa"
  }
}
```

Where `dev-mfa` is the AWS CLI profile that is configured (in `~/.aws/config`) to access the AWS development account if MFA is provided.

----

**Comment**
This setup works when deploying stacks with other resources, such as Lambda functions, S3 buckets, DynamoDB tables, etc.

**Comment by hupe1980** [comment]
Apparently the plugin was not loaded. Did you make the following entry?
```
// cdk.json
{
  "app": "npx ts-node bin/YOURAPP.ts",
  "plugin": ["cdk-multi-profile-plugin"]
}
```

**Comment by matsev** [comment]
Thanks for pointing this out. Previously, my `cdk.json` had only the `app` entry and somehow a managed to get this work without `plugin`. I know I have used the `--plugin` CLI option at some earlier point, but that was some time ago. What confused me is that I have called the `$ npx cdk synth` multiple times after that without any problem, but as soon as I added the `new VPC(...)` I encountered the credentials problem mentioned above.
